pkg/valueset/postgresql_driver.go

```diff
@@ -0,0 +1,58 @@
+package valueset
+
+import (
+	"fmt"
+
+	"github.com/jackc/pgx/v5"
+	_ "github.com/jackc/pgx/v5/stdlib"
+)
+
+// match (n:OMOPConcept) return n
+func GetResults(cfg Config, queryParams map[string]string) (interface{}, error) {
```

Comment:
Instead of config, pass the Config.Database whose adapter is pgx.

Return map[string]string